test/button.test.tsx

```diff
@@ -1,11 +1,269 @@
-import { test, Page} from "bigtest";
-import { Button as btButton } from "../src/index";
-import { Button as MyButton } from "@material-ui/core";
-import { render } from "./helpers";
+import { test, Page, HTML, Link } from "bigtest";
+import { Button as Interactor, including, not, or } from "../src/index";
+import { Button as MuiButton } from "@material-ui/core";
+import { containing } from './matchers';
+import { render, buttonClasses } from "./helpers";
+
+const button = Interactor();
+
+const Span = HTML.extend('span').selector('span');
 
 export default test("Button")
- .step(Page.visit("/"))
- .step("rendering a button", async() => {
-   render(<MyButton>Click me</MyButton>)
- })
- .assertion(btButton("CLICK ME").exists())
\ No newline at end of file
+  .step(Page.visit("/"))
+  .child("rendering a button", (test) => test
+    .step("render button", async() => {
+      await render(<MuiButton>My Button</MuiButton>)
+    })
```

Comment:
@cowboyd is this for all Interactors or just this instance?

Reply:
I'd say this would be a helper we could use in all tests. I'm just making a note here so that we can do the work in the future. It shouldn't affect this PR.